src/components/Collapsible/tests/Collapsible.test.tsx

```diff
@@ -5,18 +5,6 @@ import Collapsible from '../Collapsible';
 describe('<Collapsible />', () => {
   const ariaHiddenSelector = '[aria-hidden=true]';
 
-  it('does not render its children and indicates hidden with aria-hidden', () => {
```

Comment:
We keep the collapsed content in the DOM now for our animation. Does one see a problem with that?